README.md

```diff
@@ -1,12 +1,12 @@
 # Covered By
 
-This extension adds a field to indicate the treaties, laws, regulations or other instruments that the contracting process is covered by.
+This extension adds a field to indicate the treaties that the contracting process is covered by.
 
 ## Usage
 
-A treaty, law, regulation or other instrument may require that a buyer indicate that a contracting process is covered by it.
+A treaty may require for a member to indicate that a contracting process is covered by it. The `tender.coveredBy` field should be used to meet such requirements.
```

Comment:
Remove "for". At OCP, several people use Grammarly (there's a free option) to check for these types of errors.

For describing reality, we avoid normative words (may, should, must) and instead use their [synonyms](https://tools.ietf.org/html/draft-hansen-nonkeywords-non2119-04#page-3) (in this case, "can" instead of "may").